lib/custom/src/MShop/Service/Provider/Payment/PaypalPlus.php

```diff
@@ -0,0 +1,303 @@
+<?php
+
+/**
+ * @license LGPLv3, http://opensource.org/licenses/LGPL-3.0
+ * @copyright Aimeos (aimeos.org), 2015-2020
+ * @package MShop
+ * @subpackage Service
+ */
+
+
+namespace Aimeos\MShop\Service\Provider\Payment;
+
+
+use Omnipay\Omnipay as OPay;
+use Aimeos\MShop\Order\Item\Base as Status;
+
+/**
+ * Payment provider for payment gateways supported by the PaypalPlus library.
+ *
+ * @package MShop
+ * @subpackage Service
+ */
+class PaypalPlus
+extends    \Aimeos\MShop\Service\Provider\Payment\OmniPay
```

Comment:
There's a coding style issue, tab first then `extends`, then space, then the rest

Reply:
for coding style .. 
I have set my vs code to to run phpcs and phpcbf , but as you see it still should improved .. 
so do you recommend any "standard" for phpcs to handle the style ?
or just use online tool like : https://www.tutorialspoint.com/online_php_formatter.htm .. ?
I will correct the rest of notes after testing them .. 
does

Reply:
This is the Aimeos PHPCS ruleset:
https://github.com/aimeos/aimeos-core/blob/master/misc/coding/Aimeos/ruleset.xml